docs/changelog.md

```diff
@@ -3,6 +3,60 @@
 Maintenance branches are merged before each new release. This change log is
 ordered chronologically, so each release contains all changes described below it.
 
+# v0.23.19-RC1 (2023-02-20)
+
+This release candidate updates to Cats Effect v3.5.0-RC2, which brings [major changes](https://github.com/typelevel/cats-effect/releases/tag/v3.5.0-RC1), as well as FS2 v3.7.0-RC1. Otherwise there are no significant changes in http4s itself.
+
+## What's Changed
+
+### http4s-core
+
+* Update nscplugin, sbt-scala-native, ... to 0.4.10 in series/0.23 by @http4s-steward in https://github.com/http4s/http4s/pull/6946
+* Update scala3-library, ... to 3.2.2 in series/0.23 by @http4s-steward in https://github.com/http4s/http4s/pull/6947
```

Comment:
I usually bury this one in behind-the-scenes unless the minor version bumps, but it's no big deal.  I think the native and scalajs are notable.